File: JSBASIC/79_valid_triangle.js
// 79	79_valid_triangle.js	Check whether a triangle is valid given its three sides.

let a = 5, b = 7, c = 10;
console.log("Sides:", a, b, c);
if (a + b > c && b + c > a && a + c > b) {
    console.log("Valid Triangle.");
} else {
    console.log("NOT a Valid Triangle.");
}

File: JavaBasic/79_valid_triangle.java
// 79	79_valid_triangle.java	Check whether a triangle is valid given its three sides.
public class _79_valid_triangle {
    public static void main(String[] args) {
        int a = 5, b = 7, c = 10;
        System.out.println("Sides: " + a + ", " + b + ", " + c);
        if (a+b > c && b+c > a && a+c > b) System.out.println("Valid Triangle.");
        else System.out.println("NOT a Valid Triangle.");
    }
}
